README.md

```diff
@@ -44,25 +44,25 @@ The following chart shows the time it takes to run the 22 TPC-H queries against
 using a single executor with 8 cores. See the [Comet Benchmarking Guide](https://datafusion.apache.org/comet/contributor-guide/benchmarking.html)
 for details of the environment used for these benchmarks.
 
-When using Comet, the overall run time is reduced from 649 seconds to 433 seconds, a 1.5x speedup, with some queries
-showing a 2x-3x speedup.
+When using Comet, the overall run time is reduced from 616 seconds to 379 seconds, a 1.62x speedup, with query 1
```

Comment:
I ran with 5 iterations instead of 3 to get more stable results and re-ran the spark benchmark as well

Reply:
wondering should we have a table with times for last 5 releases?

Reply:
it would be good to show progress somewhere. Perhaps there should be a section in the documentation showing progress over time? We could also show progress in the blog post for this release.